src/phylo/distances.jl

```diff
@@ -170,7 +232,58 @@ end
 
 # K80 Distance computation.
 
+<<<<<<< HEAD
```

Comment:
This is not problematic for me at all, in fact we encourage such participation! :)